applications/better_craigslist.md

```diff
@@ -0,0 +1,153 @@
+# Enhancing zkEmail with Plonky3: Advanced Features for Decentralized Systems
+## [Section 1] Project Information
+
+Project Name: Enhancing zkEmail with Plonky3: Advanced Features for Decentralized Systems
+Payment Details: 0x0ed18cFf1e16Db3f8b76D05c84182E4849ab03D4
+Total Amount Requested: 15,000 USDC
+
+## [Section 2] Project Overview :page_facing_up:
+
+Brief Description:
+This project aims to enhance zkEmail by developing new features and improvements using Plonky3, specifically tailored for decentralized systems with a focus on C2C marketplaces. We will expand zkEmail's capabilities to address data integrity, censorship resistance, and privacy, while pushing the boundaries of Plonky3's application in practical scenarios.
+
+Core Idea:
+The primary goal is to extend zkEmail's functionality by implementing the following use cases:
+
+- Enhanced data integrity verification for email attachments using Plonky3
+  - Experiment with following attachment formats: PDF, DOCX, PNG, JPEG
+    - Convert attachments to binary
+    - Hash the binary data (SHA-256)
+    - Write circuit in plonky3 to generate proof & verifier (input: computed_hash, generated_hash)
+    
+- Censorship-resistant email content verification with Plonky3's recursive proofs
+  - Hash the email content
+  - Write circuit to generate proof of the email content
+  - Use plonky3's recursive proof to chain proofs of emails 
+  - Chain the proofs to generate a final proof and verifier 
+
+- Privacy-preserving reputation systems based on email interactions using Plonky3's lookup arguments
+  - Gather interaction data (data in email interactions such as the number of emails sent/received, response times, content length, etc.).
```

Comment:
Regarding to gather interaction data, we only able to gather it as long as we have access to the mail server, the proposal is missing the clarification for this in reputation feature.

Will reputation calculation algorithm be available publicly? If it available publicly how can we make sure adversaries won't able to trick the algorithm?

Reply:
For the reputation system, we just removed it from the proposal as after some research we realised the work timeframe would be unrealistic for having it in this same proposal. Therefore, we chose to take it out and we intend to submit as a proposal in a later round, though we have the algorithm hashed out.
With this removed we can focus more on the usage of plonky3 to implement this new feature for it and also improve some of the circuits that are in circom.